config/packages/prod/monolog.yaml

```diff
@@ -20,27 +20,36 @@ monolog:
         - swp_asset_download
         - swp_content_push
         - swp_image_conversion
+        - swp_rule
+        - swp_validators
     handlers:
-        main:
-            type:         fingers_crossed
-            action_level: error
-            handler:      nested
-            excluded_http_codes: [404]
-        nested:
-            type:  service
-            id:    monolog.gelf_handler
-            level: error
-            channels: ["!swp_send_webhook", "!swp_content_push", "!swp_image_conversion", "!swp_asset_download", "!swp_rule", "!swp_validators"]
         console:
             type:  console
-        gelf:
-            level: debug
-            type: service
-            id: monolog.gelf_handler
+        custom_channels:
+            type: fingers_crossed
```

Comment:
this may be changed to `buffer` maybe but `fingers_crossed` with this `action_level` also works.